lesson_03/slices/slices.go

```diff
@@ -0,0 +1,23 @@
+package slices
+
+func MaxLength(slice []string) string {
```

Comment:
not meaningfull name for function, longestWorld or something like that, also not shure if it will work correctly. Try please with next array []string{"вася","hello"}